.github/workflows/release.yaml

```
@@ -17,6 +17,7 @@ jobs:
         - aarch64-apple-darwin
         - aarch64-unknown-linux-musl
         - armv7-unknown-linux-musleabihf
+        - arm-unknown-linux-musleabihf
```

I like to keep these sorted:

```suggestion
        - arm-unknown-linux-musleabihf
        - armv7-unknown-linux-musleabihf
```